src/panels/lovelace/editor/config-elements/hui-iframe-card-editor.ts

```diff
@@ -19,6 +18,18 @@ const cardConfigStruct = assign(
   })
 );
 
+const SCHEMA: HaFormSchema[] = [
+  { name: "title", selector: { text: {} } },
+  {
+    name: "",
+    type: "grid",
+    schema: [
+      { name: "url", required: true, selector: { text: {} } },
+      { name: "aspect_ratio", selector: { text: {} } },
```

Comment:
Do we have a placeholder for the aspect ratio that we use when it's not filled in ? That also helps the user to show what kind of values we expect.

Reply:
And maybe one day we make a visual aspect ratio selector.

Reply:
The default is 50%. Which is shown to the user in the editor